elvis.config

```diff
@@ -3,14 +3,28 @@
   elvis,
   [
    {config,
-    [#{dirs => ["apps/*/src", "apps/*/test", "test/ejabberd_tests/tests"],
+    [#{dirs => ["apps/*/src"],
        filter => "*.erl",
        ruleset => erl_files,
        rules => [{elvis_style, line_length, #{limit => 100,
                                               skip_comments => false}},
 		 {elvis_style, dont_repeat_yourself, #{min_complexity => 20}}
 		]
-      }
+      },
+     #{dirs => ["apps/*/test", "test/ejabberd_tests/tests"],
+       filter => "*.erl",
+       ruleset => erl_files,
+       rules => [{elvis_style, line_length, #{limit => 100,
+                                              skip_comments => false}},
+                 {elvis_style, dont_repeat_yourself, disable},
```

Comment:
I don't like this. We should aim at improving test's code and one way of doing it is by eliminating duplicated code.